src/test/java/org/kcctl/support/KcctlCommandContext.java

```diff
@@ -30,4 +32,19 @@ public void reset() {
         output.getBuffer().setLength(0);
         error.getBuffer().setLength(0);
     }
+
+    /**
+     * Run a command with the given arguments and verify that it has exited successfully
+     * @param args the arguments with which the command should be invoked
+     */
+    public void runAndCheckExitCode(String... args) {
```

Comment:
"check exit code" to me suggests that I could specify which code I'm expecting. But given we expect that the command succeeds, making this more explicit in the name would be nice. How about `runAndEnsureExitCodeOk()`, `runAndEnsureStatusOk()`, `runAndEnsureSuccessfulExecution()`, or something along those lines?

Reply:
Thanks, I agree that it's a little unclear. Went with `runAndEnsureExitCodeOk`.

Reply:
Perfect, thx!